framework/src/modules/token/method.ts

```diff
@@ -703,8 +750,38 @@ export class TokenMethod extends BaseMethod {
 		this.events.get(TokenIDSupportedEvent).log(methodContext, tokenID);
 	}
 
-	public async removeSupportTokenID(methodContext: MethodContext, tokenID: Buffer): Promise<void> {
-		await this.stores.get(SupportedTokensStore).removeSupportForToken(methodContext, tokenID);
```

Comment:
if we don't need store function like `removeSupportForToken` we can remove and move the unit tests to method

Reply:
I am not sure if it's needed or not but had to rename it from `removeSupportTokenID` to `removeSupport` as based on our prev discussion, the idea was to align the function names as well.

![image](https://github.com/LiskHQ/lisk-sdk/assets/10962699/e1c66a65-3c93-4bab-8ccf-d66cb59f8db8)
links to
![image](https://github.com/LiskHQ/lisk-sdk/assets/10962699/766949e8-52cc-467c-861f-84120a678b17)

Reply:
i meant `await this.stores.get(SupportedTokensStore).removeSupportForToken` this function on `Store` which is robably not used anymore